tasks/main.yml

```diff
@@ -45,10 +45,21 @@
     regexp: SSH_AUTH_SOCK
     line: Defaults env_keep += "SSH_AUTH_SOCK"
 
-- name: install base packages
+- name: install 16.04-specific base packages
```

Comment:
python-software-properties isn't specific to 16.04, it's been around for a long time.  We'll need it for any install up through 16.04, I would think.

Reply:
Ah, great point!